projects/amulet-protocol/index.js

```diff
@@ -19,10 +19,22 @@ async function tvl() {
     const accountSolBalances = await Promise.all(accountList.map(async(x) => new BN((await connection.getAccountInfo(new PublicKey(x))).lamports)))
     const solAmount = [new BN(posSolVirtualAmount), new BN(liqSolAmount), new BN(totalActiveBalance), ...accountSolBalances]
 
-    const tvl = solAmount.reduce((acc,x) => acc.add(x), new BN(0)).div(new BN(1000000000)).toNumber()
+    const tvlSOL = solAmount.reduce((acc,x) => acc.add(x), new BN(0)).div(new BN(1000000000)).toNumber()
+
+    const stakingInstanceState = await connection.getAccountInfo(new PublicKey("HNhPNHkp3RobeJzepNzyVyewtAaoF3QCCvtBTxKJVnRX"));
+    const liqStakedAmtSOLAmount = stakingInstanceState.data.slice(203, 211);
```

Comment:
sorry, it is not very clear, what does this refer to?

Reply:
Hi this is amtSOL amount staked in our single-sided amtSOL-aUWT pool